core/consensus/babe/impl/block_executor_impl.cpp

```diff
@@ -64,6 +69,13 @@ namespace kagome::consensus {
     BOOST_ASSERT(babe_util_ != nullptr);
     BOOST_ASSERT(offchain_worker_api_ != nullptr);
     BOOST_ASSERT(logger_ != nullptr);
+
+    // Register metrics
+    metrics_registry_->registerHistogramFamily(
+        kBlockExecutionTime, "Time taken to verify and import blocks");
+    metric_block_execution_time_ = metrics_registry_->registerHistogramMetric(
+        kBlockExecutionTime,
+        {0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10});
```

Comment:
I see this list the second time. Two suggestions:
* name it (the whole list) to avoid the practice of magic constants
* make it somewhere top-level predefined constant

Reply:
+
Maybe we can create some metrics_common.hpp file with such constants

Reply:
Some intervals set makes sense for one metric and inappropriate for others. I think, almost all metrics should be clean customized, in-place.